carma_cooperative_perception/config/cp_sdsm_to_detection_list_node.yaml

```diff
@@ -24,9 +24,9 @@ twist_covariance_z: 0.005  # Z-axis velocity uncertainty
 twist_covariance_yaw: 0.005  # Yaw angular velocity uncertainty
 
 # True to enable adding offsets to the pose of the detected object generated from the SDSM
-adjust_pose: true
+adjust_pose: False
```

Comment:
This value is already captured by the release/tempest. We don't need it automatically turned on anymore as SDSM service is reliable.